### ISSUE TITLE
change folder name

### ISSUE THREAD

**Issue by jhardy1974** [pwnall/dropship-chrome#3]
is there a way to change the folder name that the files download to

**Comment by pwnall** [comment]
The extension uses the "App Folder" permission, so it is restricted to a folder in your Dropbox.

If you want to upload to `/some/path` in your Dropbox, you can do the following:
- Fork the repository
- Create a new Dropbox API app -- https://www.dropbox.com/developers/apps/create
- Answer the questions as follows:
  - type of app: `Dropbox API app`
  - type of data: `Files and Datastores`
  - private folder: `No`
  - type of files: `All file types`
- Copy the app key here: https://github.com/pwnall/dropship-chrome/blob/master/src/coffee/event_page.coffee#L267
- Change this line: https://github.com/pwnall/dropship-chrome/blob/master/src/coffee/options.coffee#L84
  to
  `"/some/path/" + uploadPath`
- Enable `Developer mode` at chrome://extensions/ and load the unpacked extension in the `build/` directory

Does this work for you?

**Comment by jhardy1974** [comment]
yep, that's what I was wondering, I'm a bit of a nut about my folders being in specific order
